src/extensions/sonarqube/vss-extension.json

```diff
@@ -4,8 +4,8 @@
   "name": "SonarQube Server",
   "version": "7.1.1",
   "branding": {
-    "color": "rgb(67, 157, 210)",
-    "theme": "dark"
```

Comment:
Yes previously we were showing a dark-blue background so we used `theme=dark` to render the text in white, but now because we switched to a white-ish background we need the text back to dark (like the SQ Cloud extension)